offline-tools/copyrighter/pkg/cmd/galasacopyrighter_test.go

```diff
@@ -190,3 +190,89 @@ func TestClosingCommentIsBeforeOpeningCommentFailsWithError(t *testing.T) {
 	assert.NotNil(t, err)
 	assert.Contains(t, err.Error(), "closing comment marker found before the starting comment marker")
 }
+
+func TestCommentIsPresentButDoesNotIncludeCopyright(t *testing.T) {
+	// Given..
+	var input = `
+/*
+ * Hello, World
+ */
+ package mypackage`
+
+	// When...
+	output, _ := setCopyright(input)
+
+	// Then..
+	assert.NotNil(t, output)
+	assert.Contains(t, output, "* Hello, World")
+	assert.Contains(t, output, "* Copyright contributors to the Galasa project")
+	//println(output)
+}
+
+func TestCommentIsPresentButDoesNotIncludeCopyrightLeadingTextPreserved(t *testing.T) {
+	// Given..
+	var input = `leading text here
+/*
+ * Hello, World
+ */
+ package mypackage`
+
+	// When...
+	output, _ := setCopyright(input)
+
+	// Then..
+	assert.NotNil(t, output)
+	assert.Contains(t, output, "leading text here")
+	assert.Contains(t, output, "* Hello, World")
+	assert.Contains(t, output, "* Copyright contributors to the Galasa project")
+}
+
+func TestCommentDoesNotIncludeCopyrightButStartsWithClosingComment(t *testing.T) {
+	// Given..
+	var input = `
+*/
+ * Hello, World
+ /*
+ package mypackage`
+
+	// When...
+	_, err := setCopyright(input)
+
+	// Then..
+	//println(output)
+	assert.NotNil(t, err)
+	assert.Contains(t, err.Error(), "closing comment marker found before the starting comment marker")
+}
+
+func TestCommentIsPresentAndIncludesCopyright(t *testing.T) {
+	// Given..
+	var input = `
+/*
+ * Copyright contributors to the Galasa project
+ */
+ package mypackage`
+
+	// When...
+	output, _ := setCopyright(input)
+
+	// Then..
+	assert.NotNil(t, output)
+	assert.Contains(t, output, "* SPDX-License-Identifier: EPL-2.0")
```

Comment:
Lets check that the output doesn't have `Copyright contributors to the Galasa project` inside ?